Gopkg.toml

```diff
@@ -124,3 +124,7 @@ required = ["github.com/wadey/gocovmerge"]
   [[prune.project]]
     name = "github.com/docker/cli"
     non-go = false
+
+[[constraint]]
```

Comment:
Is it really a constraint? Could this be an override?

Reply:
Actually I let `dep ensure -add github.com/stretchr/testify ` update the file. It's a new test dependency of the project and as far as i understand dep I think it should be a constraint.